KlockanAPI/src/KlockanAPI.Infrastructure/Repositories/MeetingRepository.cs

```diff
@@ -0,0 +1,22 @@
+using Microsoft.EntityFrameworkCore;
+using KlockanAPI.Domain.Models;
+using KlockanAPI.Infrastructure.Data;
+using KlockanAPI.Infrastructure.Repositories.Interfaces;
+
+
+namespace KlockanAPI.Infrastructure.Repositories;
+
+public class MeetingRepository : IMeetingRepository
+{
+    private readonly KlockanContext _context;
+
+    public MeetingRepository(KlockanContext context)
+    {
+        _context = context;
+    }
+
+    public async Task<IEnumerable<Meeting>> GetAllAsync()
+    {
+        return await _context.Meetings.Include(m => m.Classroom).ToListAsync();
+    }
+}
```

Comment:
This should also return, in case it exists, the trainer associated to that meeting, making the relationship with ClassroomUser, Users and Roles.

Reply:
I mentioned the the last week that i will comment that there not created seeders for trainers they will be implemented when trainers seeders are created(out of my scope), i forgot to leave a comment on that.